### ISSUE TITLE
`pip install trinity` gets wrong dependency version

### ISSUE THREAD

**Issue by zjiekai** [ethereum/trinity#1339]
* Trinity version: 0.1.0a30
* Python version (output of `python --version`): Python 3.6.8

### What is wrong?

Installed trinity with `pip install trinity`, got dependency `lahja 0.14.6` requiring `trio 0.13`.

~~Running trinity outputs a lot of `DeprecationWarning`.~~

`trinity attach` can not run.

```
Unhandled exception in event loop:                                                                                                 
  File "uvloop/cbhandles.pyx", line 64, in uvloop.loop.Handle._run                   
  File "/home/zjk/py3env/trinity2/lib/python3.6/site-packages/prompt_toolkit/eventloop/utils.py", line 69, in schedule                       
    if not loop2._ready:  # type: ignore                                                                                                     
                                                                                                                                             
Exception 'Loop' object has no attribute '_ready'   
```

This seems to be another issue.

### How can it be fixed

`pip install lahja==0.14.5`

**Comment by cburgdorf** [comment]
Hi @zjiekai yes, it turns out that we accidentally released a `patch` release of `lahja` that should have been a `minor` release.

@pipermerriam I noticed that you released `0.15.0` but should we know just nuke `0.14.6

![image](https://user-images.githubusercontent.com/521109/69946671-65254480-14ec-11ea-8547-1a9f07383bb8.png)

I was about to do it but are looking for a final approval. It seems like the easiest option to unbreak past Trinity releases.

**Comment by zjiekai** [comment]
Oh the `attach` issue is getting fixed 

https://github.com/ethereum/trinity/pull/1343

**Comment by carver** [comment]
The attach issue is fixed on master, but since it's a busted dependency, be sure to pip install and make sure that `ipython` is lower than v7.10.

If that completely fixes your problem, then please close. If not, please update the issue with the new warnings/errors you see.

**Comment by cburgdorf** [comment]
@carver these were two different issues. However, I just deleted the lahja `0.14.6` release (which was re-released as `0.15.0` so this should unbreak the past two Trinity releases (which accidentally broke because lahja `0.14.6` was a breaking change).

@zjiekai I'm going to close this. Feel free to re-open if the error somehow persists. Please note though that you might have to clean all you dependencies and re-install them.
